openedx/stanford/cms/envs/common.py

```diff
@@ -50,6 +50,7 @@
 })
 INSTALLED_APPS += (
     'openedx.stanford.djangoapps.course_utils',
+    'openedx.stanford.djangoapps.student_utils',
```

Comment:
You could have prevented the tests from running in the cms suite by using:

from django.conf import settings

@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
class BulkUserActivateTests(TestCase):

Reply:
@dcadams Kevin suggested that too..
We shouldn't disable tests just because they fail. This command was designed w/out any dependencies on either LMS or CMS, so it should be able to run properly in both; our tests should confirm this.